test cases/common/218 include_dir dot/src/meson.build

```diff
@@ -1,6 +1,21 @@
+# test to_list() for include_directories defined in the parent meson.build file
+source_dir = meson.project_source_root()
+build_dir = meson.project_build_root()
+
+expected = [source_dir / '.', build_dir / '.']
+message('actual: @0@'.format(inc.to_list()))
+message('expected: @0@'.format(expected))
+
+# Use fs.as_posix() because / operator replaces \ with / in the path, but
+# full_path() method is not doing that. This is a pretty inconsistent across all
+# Meson APIs.
```

Comment:
I think this is not the right place to complain about inconsistencies in meson. I would rather something like:

```
Use fs.as_posix() to ensure the results are the same across platforms.
```

Reply:
I wasn't complaining, sorry! I just copy/pasted the same comment from test case #74. I searched to how other test cases addressed the issue and that was the first test case I found. I can change the comment to the language you suggest as well as change the comment in test case 74 to use the same language.

Reply:
Done. Used the suggested language.